src/server/src/main/java/io/cassandrareaper/resources/RepairScheduleResource.java

```diff
@@ -76,6 +75,160 @@ public RepairScheduleResource(AppContext context) {
     this.repairRunService = RepairRunService.create(context);
   }
 
+  /**
+   * Validate the set of parameters used to patch a RepairSchedule.
+   *
+   * @return List of invalid parameters, empty if all are valid
+   */
+  protected static List<String> validateRepairPatchParams(
+          String owner,
+          String repairParallelism,
+          Double intensity,
+          Integer scheduleDaysBetween,
+          Integer segmentCountPerNode
+  ) {
+    List<String> invalidParams = Lists.newArrayList();
+
+    boolean ownerValid = owner != null && !owner.isEmpty();
+
+    boolean repairParallelismValid = repairParallelism != null && !repairParallelism.isEmpty();
+    if (repairParallelismValid) {
+      try {
+        repairParallelismValid = RepairParallelism.valueOf(repairParallelism.toUpperCase()) != null ? true : false;
+      } catch (Exception e) {
+        repairParallelismValid = false;
+      }
+    }
+    boolean intensityValid = intensity != null && intensity >= 0.0D && intensity <= 1.0D;
+    // TODO What are good valid ranges for these values?
+    boolean scheduleDaysBetweenValid = scheduleDaysBetween != null && scheduleDaysBetween > 0;
```

Comment:
Another validation question here for you @adejanovski what are reasonable boundaries for these?

Reply:
The interval can be 0, which means repair will run continuously, restarting right after it's finished. So I'd say `0 <= scheduleDaysBetweenValid <= 31`.
I can't think of a use case where you'd want to schedule a repair over a monthly basis (which is already super high if you ask me, the default being at least every 10 days).

Reply:
Perfect, thanks.  I've updated the new validations with the rules you mention.